config/v1/types_cluster_version.go

```diff
@@ -123,14 +123,25 @@ type ClusterVersionStatus struct {
 	// +optional
 	Conditions []ClusterOperatorStatusCondition `json:"conditions,omitempty"`
 
-	// availableUpdates contains the list of updates that are appropriate
-	// for this cluster. This list may be empty if no updates are recommended,
-	// if the update service is unavailable, or if an invalid channel has
-	// been specified.
+	// availableUpdates contains updates recommended for this
+	// cluster. Updates which appear in conditionalUpdates but not in
+	// availableUpdates may expose this cluster to known issues. This list
+	// may be empty if no updates are recommended, if the update service
+	// is unavailable, or if an invalid channel has been specified.
 	// +nullable
 	// +kubebuilder:validation:Required
 	// +required
 	AvailableUpdates []Release `json:"availableUpdates"`
+
+	// conditionalUpdates contains the list of updates that may be
+	// recommended for this cluster if it meets specific required
+	// conditions. Consumers interested in the set of updates that are
+	// actually recommended for this cluster should use
+	// availableUpdates. This list may be empty if no updates are
+	// recommended, if the update service is unavailable, or if an empty
+	// or invalid channel has been specified.
+	// +optional
+	ConditionalUpdates []ConditionalUpdate `json:"conditionalUpdates,omitempty"`
```

Comment:
a list-type here?

Reply:
`+listType=atomic` in 98cb02fd -> 39ecc131